src/Tribe/Attendee_Registration/Template.php

```diff
@@ -58,15 +60,28 @@ public function setup_context( $posts ) {
 			return $posts;
 		}
 
+		/*
+		 * Early bail:
+		 * We are on the AR page, but we have the shortcode in the content,
+		 * so we don't want to spoof this page.
+		 */
+		if ( is_array( $posts ) && ! empty( $posts ) ) {
+			if ( $posts[0] instanceof WP_Post ) {
+				if ( has_shortcode( $posts[0]->post_content, "tribe_attendee_registration" ) ) {
```

Comment:
:no_entry_sign: String "tribe_attendee_registration" does not require double quotes; use single quotes instead

<sub>hash: 961656190d56c435d2ccc17a2ae8b838</sub>

Reply:
```suggestion
				if ( has_shortcode( $posts[0]->post_content, 'tribe_attendee_registration' ) ) {
```